imports/startup/server/migrations.js

```diff
@@ -445,6 +448,25 @@ Migrations.add({
   },
 });
 
+Migrations.add({
+  version: 25,
+  name: 'Parcel.type and Membership.rank becomes simple text field',
+  up() {
+    Parcels.find({ type: { $exists: true } }).forEach((p) => {
+      const language = p.community().settings.language;
+      const type = TAPi18n.__(`schemaParcels.type.${p.type}`, {}, language);
+      Parcels.update(p._id, { $set: { type } }, { selector: { category: '@property' } }, { validate: false });
+    });
+    officerRoles.forEach(role => {
+      Memberships.find({ role, rank: { $exists: true } }).forEach((m) => {
+        const language = m.community().settings.language;
+        const rank = TAPi18n.__(`schemaMemberships.rank.${m.rank}`, {}, language);
+        Memberships.update(m._id, { $set: { rank } }, { selector: { role } }, { validate: false });
+      });
+    });
+  },
+});
+
```

Comment:
Ha a community.parcels-en tároljuk a parceltype-okat, akkor azokat is migrálni kéne. Plusz az albetét tábla showAllParcels-ét is máshogy kell akkor számolni, mert most a community.parcels.flat-et nézi.

Reply:
Jo eszrevetel! Nem muszaj a tipusra szamolni itt. Lehet nezni mondjuk a vezer albetetek szamat. -- Parcels.find({ leadRef: { $exists: false } }).count()